docs/UserGuide.adoc

```diff
@@ -191,7 +191,7 @@ Displays a sorted list of upcoming appointments for the patient that is associat
 If no keyword is given, all upcoming appointments will be displayed. +
```

Comment:
Please don't merge your UG. I already made the changes in PR #197

Reply:
okay